packages/components/time-select/src/time-select.vue

```diff
@@ -68,9 +73,10 @@ const compareTime = (time1: string, time2: string): number => {
   return minutes1 > minutes2 ? 1 : -1
 }
 const formatTime = (time: Time): string => {
-  return `${time.hours < 10 ? `0${time.hours}` : time.hours}:${
-    time.minutes < 10 ? `0${time.minutes}` : time.minutes
-  }`
+  return `${`${time.hours}`.padStart(2, '0')}:${`${time.minutes}`.padStart(
```

Comment:
For using this method about 4 times in this same file, you should consider use a function to encapsulate this:

```ts
function padTime(time: number | string) {
  return `${time}`.padStart(2, 0)
}
```

Then you use it like:

```ts
return `${padTime(time.hours)}:${padTime(time.minutes)}`
```

Reply:
ok